plugins/vendor/webex/services/WebexVendorService.php

```diff
@@ -299,7 +299,7 @@ public function submitRegistrationAction($accountId, $integrationSetting)
 		{
 			$webexIntegration->unsetCategory();
 		}
-		
+		$integrationSetting->enableMeetingChat = true;
```

Comment:
I think that the correct thing is to define to getEnableMeetingChat to true by default, else each time someone press submit it will enable (even if he explicitly set it in the API to false)